apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.jsx

```diff
@@ -50,6 +50,7 @@ import SwapFormSummary from "./FormSummary";
 import SwapFormRates from "./FormRates";
 import { DEX_PROVIDERS } from "~/renderer/screens/exchange/Swap2/Form/utils";
 import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
+import debounce from "lodash/debounce";
```

Comment:
Across the app we are using 
import { useDebounce } from "@ledgerhq/live-common/hooks/useDebounce";
with 500 ms.